crates/bevy_light/src/directional_light.rs

```diff
@@ -240,3 +240,38 @@ pub fn update_directional_light_frusta(
             .collect();
     }
 }
+
+/// This component marks a [`DirectionalLight`] entity for changing the size and intensity of the sun disk.
+#[derive(Component, Clone)]
+#[require(DirectionalLight)]
+pub struct SunLight {
+    /// The angular size (diameter) of the sun disk in radians as observed from Earth.
+    pub angular_size: f32,
+    /// Multiplier applied to the brightness of the sun disk in the sky.
+    ///
+    /// `0.0` disables the sun disk entirely while still
+    /// allowing the sun's radiance to scatter into the atmosphere,
+    /// and `1.0` renders the sun disk at its normal intensity.
+    pub intensity: f32,
+}
+
+impl SunLight {
+    pub const SUN: SunLight = SunLight {
```

Comment:
typing out `SunLight:SUN` or `SunDisk:SUN` feels a bit awkward, but I don't have a better idea. maybe `SunDisk:EARTH` ? since this is exactly 1 AU away? I am adding martian atmosphere eventually so there can be a `SunDisk:MARS` too that has different values.

Reply:
Maybe `SunDisk::VIEWED_FROM_EARTH`